patterns/discount-banner.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Title: Discount Banner
+ * Slug: woocommerce-blocks/discount-banner
+ * Categories: WooCommerce
+ */
+?>
+
+<!-- wp:columns {"verticalAlignment":"center"} -->
+<div class="wp-block-columns are-vertically-aligned-center">
+	<!-- wp:column {"verticalAlignment":"center","width":"400px","style":{"color":{"background":"#254094"},"spacing":{"padding":{"top":"25px","right":"40px","bottom":"40px","left":"40px"}}}} -->
+	<div class="wp-block-column is-vertically-aligned-center has-background" style="background-color:#254094;padding-top:25px;padding-right:40px;padding-bottom:40px;padding-left:40px;flex-basis:400px">
+		<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"500","fontSize":"45px"}},"textColor":"base"} -->
+		<p class="has-base-color has-text-color" style="font-size:45px;font-style:normal;font-weight:500">UP TO</p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:paragraph {"style":{"color":{"text":"#fdf251"},"typography":{"fontStyle":"normal","fontWeight":"800","fontSize":"90px","lineHeight":"0"}}} -->
+		<p class="has-text-color" style="color:#fdf251;font-size:90px;font-style:normal;font-weight:800;line-height:0">
+			40% off
+		</p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"300","fontSize":"35px"}},"textColor":"base"} -->
+		<p class="has-base-color has-text-color" style="font-size:35px;font-style:normal;font-weight:300">Select product</p>
```

Comment:
Minor, but design says "Select productS":

```suggestion
		<p class="has-base-color has-text-color" style="font-size:35px;font-style:normal;font-weight:300">Select products</p>
```